ndc-reference/tests/query/predicate_with_exists_in_nested_collection/request.json

```diff
@@ -0,0 +1,63 @@
+{
+    "$schema": "../../../../ndc-models/tests/json_schema/query_request.jsonschema",
+    "collection": "institutions",
+    "arguments": {},
+    "query": {
+        "fields": {
+            "id": {
+                "type": "column",
+                "column": "id"
+            },
+            "name": {
+                "type": "column",
+                "column": "name"
+            },
+            "staff": {
+                "type": "column",
+                "column": "staff",
+                "arguments": {
+                    "limit": {
+                        "type": "literal",
+                        "value": null
+                    }
+                }
+            }
+        },
+        "predicate": {
+            "type": "exists",
+            "in_collection": {
+                "type": "nested_collection",
+                "arguments": {
+                    "limit": {
+                        "type": "literal",
+                        "value": null
+                    }
+                },
+                "column_name": "staff"
+            },
+            "predicate": {
+                "type": "binary_comparison_operator",
+                "column": {
+                    "type": "column",
+                    "name": "last_name"
+                },
+                "operator": "like",
+                "value": {
+                    "type": "scalar",
+                    "value": "s"
+                }
+            }
+        }
+    },
+    "collection_relationships": {
+        "author_articles": {
```

Comment:
Am I missing something, or is this relationship definition not required for this request?

Reply:
Yes, thanks.